### PR TITLE
README.rst: Add screenshot of the demo page.

### PR DESCRIPTION
**Pull request by marian-ald**
Closes https://github.com/coala/coala-html/pull/101

### CODE REVIEW

**Comment by Mixih** [comment]
You never use a pull request for the closes section. Instead, you whould link to the parent issue.

File: README.rst [hunk 1]
```diff
@@ -44,6 +44,10 @@ DEMO
 ----
 GIF_.  Live_.
 
+Demo page
```
Comment (aptrishu) [inline README.rst]: Every heading is in Block letters. Change your heading to block letters. Lets see how it looks.

**Comment by aptrishu** [comment]
Change the Closes section 
`Closes {url of the issue you're working on, not the PR}`

Also you made two commits, squash them.
Use `git commit -a --amend` from next time. and force push that.

**Comment by marian-ald** [comment]
How much below? Because there are GETTING INVOLVED and LICENSE section... So shouldn't it be in the ABOUT section?

**Comment by aptrishu** [comment]
There are so many demo this, demo that in the file, doesn't look good.
CC @Mixih

**Comment by yash-nisar** [comment]
I meant below ABOUT section and above PROJECT STATUS section, ie. between the ABOUT and PROJECT STATUS section. Also, since we already have a title named DEMO, renaming DEMO PAGE to something else would be a good option. :smile:

**Comment by sils** [comment]
unneeded period in commit message shortlog

**Comment by sils** [comment]
otherwise good

**Comment by aptrishu** [comment]
ack 96fc168

**Comment by aptrishu** [comment]
@rultor merge

**Pull request merged**